### ISSUE TITLE
LDT downscaling for GDAS not working correctly

### ISSUE THREAD

**Issue by dpsarmie** [NASA-LIS/LISF#66]
When using the 'Metforce temporal downscaling' option in LDT with GDAS, the GDAS reader does not properly reset and results in the same field until the secondary met forcing source resets.
This was found when trying to downscale CHIRPS data using the GDAS forcing.

**Comment by dmocko** [comment]
Is this issue now closed, with pull request #67 now closed?

**Comment by dpsarmie** [comment]
Yes, this issue is resolved.  I thought that referencing the issue in the commit message would auto-close the linked issue once the commit was merged. Maybe I did something wrong? (Or that isn't a feature on Github.)

**Comment by dmocko** [comment]
Looks like you did things correctly, and I thought the linked issue would auto-close.  Not sure what happened.  Thanks for confirming.
